src/ripple/app/misc/FeeEscalation.md

```diff
@@ -100,7 +107,7 @@ ledger from highest [fee level](#fee-level) to lowest. These transactions
 count against the open ledger limit, so the required [fee level](#fee-level)
 may start rising during this process.
 3. Once the queue is empty, or required the [fee level](#fee-level)
-jumps too high for the remaining transactions in the queue, the ledger
+rises too high for the remaining transactions in the queue, the ledger
 is opened up for normal transaction processing.
 4. A transaction in the queue can stay there indefinitely in principle,
```

Comment:
Your second paragraph is correct. The countdown is only affected when the transaction actually gets a chance to be processed.

I can see how that would seem counter intuitive, but I'm looking at it from a different angle. I'm not worried about lifespan, I'm worried about giving every transaction that's been broadcast as much chance to _eventually_ get into a ledger as possible. So if a transaction is higher priority, but is really just not likely to succeed, it's better to get rid of it, and stop wasting the resources that are being used to keep reprocessing it. On the other hand, if a transaction is low priority, and the queue is not full, but circumstances just keep lining up so that it never gets a chance, then it's not really doing any harm - it's taking up some memory, but it's not wasting time, processing, network, etc, which is more what I prioritized for. Also, there is a mechanism for that transaction to eventually be dropped - if the queue does get full of higher priority transactions, it'll get pushed off the end.